### PR TITLE
finalize release notes for v5.6.0

### PR DESCRIPTION
**Pull request by pivotal-jamie-klassen**
also documents #4446

### CODE REVIEW

File: release-notes/v5.6.0.md [hunk 1]
```diff
@@ -1,67 +1,69 @@
...
-  Defaults to Garden's range of 10.254.0.0/22. Addresses #4153.
+  It's worth noting that concourse performs a lot of checks (like A LOT). Since we're now storing checks in the database, this table will tend to grow very quickly. By default checks get gc'ed every 6 hrs, but this interval can be configured by specifying a `CONCOURSE_GC_CHECK_RECYCLE_PERIOD`. If you want to reduce the number of checks that happen, you can start making heavier use of the `webhook` endpoint to trigger checks from external sources. This allows you to significantly reduce the `check_every` interval (default 1m) for your resource without impacting the time it takes to schedule a build. 
 
-#### <sub><sup><a name="v560-note-4438" href="#v560-note-4438">:link:</a></sup></sub> feature
+  If you're interested in more detail about what changed you can have a look at the corresponding PR #4202 or the initial issue #3788
 
-* The worker name will now be added to the `data` structure that is included in log entries related to Containers & Volumes. #4438
+  This feature is off by default but can be turned on via `CONCOURSE_ENABLE_LIDAR`.
```
[the remark this answers is not in the document]
Comment (pivotal-jamie-klassen) [reply release-notes/v5.6.0.md]: @pivotal-jwinters can you make a suggestion here? My concern is kinda that both of the new checks API and the lidar functionality are in the same note... although you would understand the finer points of this better than I

File: release-notes/v5.6.0.md [hunk 2]
```diff
@@ -1,67 +1,69 @@
...
-* Fly has a new sub-command `pin-resource`. It allows you to pin a resource given a partially matched version. It can also be used to set or update a pin comment. #2702 #4417.
-
-#### <sub><sup><a name="v560-note-4481" href="#v560-note-4481">:link:</a></sup></sub> feature
-
-* @evanchaoli added `age` column to `fly workers`, #4481.
-
-#### <sub><sup><a name="v560-note-4314" href="#v560-note-4314">:link:</a></sup></sub> fix
-
-* @robwhitby fixed an issue with `fly login` where Safari would block your token from being transferred to fly #4314, #4423, #4439.
+* Fly has a new sub-command `pin-resource`, which will pin a resource (and optionally comment) given at least one field of the version to pin to #2702 #4417.
```
[the remark this answers is not in the document]
Comment (xtremerui) [reply release-notes/v5.6.0.md]: Yeah somehow we missed that one. Will add it.

File: release-notes/v5.6.0.md (hunk 1 above)
Comment (pivotal-jwinters) [inline release-notes/v5.6.0.md]:
```suggestion
  This feature is still experimental so it's off by default, but can be turned on via `CONCOURSE_ENABLE_LIDAR`.
```

File: release-notes/v5.6.0.md [hunk 3]
```diff
@@ -1,71 +1,72 @@
-#### <sub><sup><a name="v560-note-3895" href="#v560-note-3895">:link:</a></sup></sub> feature
+#### <sub><sup><a name="v560-note-4202" href="#v560-note-4202">:link:</a></sup></sub> feature
 
-* @ralekseenkov added a web runtime flag `CONCOURSE_SECRET_CACHE_DURATION_NOTFOUND` to set a
-  separate caching interval when a secret is not successfully found in the config store.
-  Defaults to 10s. Addresses #3895 #4009.
+* There is a a new experimental method of resource checking, which is off by default but can be turned on via `CONCOURSE_ENABLE_LIDAR`.
```
Comment (vito) [inline release-notes/v5.6.0.md]:
```suggestion
* There is a new experimental method of resource checking, which is off by default but can be turned on via `CONCOURSE_ENABLE_LIDAR`.
```

**Pull request merged**